src/cosmic_ray/testing.py

```diff
@@ -74,6 +74,5 @@ def run_tests(command, timeout=None):
         asyncio.set_event_loop_policy(
             asyncio.WindowsProactorEventLoopPolicy())
 
-    result = asyncio.get_event_loop().run_until_complete(
-        _run_tests(command, timeout))
+    result = asyncio.run(_run_tests(command, timeout))
```

Comment:
Because there is something wrong when children processes use this code when parent process use asyncio.run ...
Children seems to know something about its parent event loop. this is not clear for me too.